server/sonar-web/src/main/js/apps/organizations/components/MembersListItem.js

```diff
@@ -40,14 +45,41 @@ export default class MembersListItem extends React.PureComponent {
     return (
       <tr>
         <td className="thin nowrap">
-          <Avatar hash={member.avatar} size={AVATAR_SIZE} />
+          <Avatar hash={member.avatar} email={member.email} size={AVATAR_SIZE} />
         </td>
         <td className="nowrap text-middle"><strong>{member.name}</strong></td>
         {organization.canAdmin &&
           <td className="text-right text-middle">
             {translate('organization.members.x_group(s)', formatMeasure(member.groupCount, 'INT'))}
-          </td>
-        }
+          </td>}
+        {organization.canAdmin &&
+          <td className="nowrap text-middle text-right">
+            <div className="dropdown">
+              <button className="dropdown-toggle little-spacer-right" data-toggle="dropdown">
+                <i className="icon-settings" />
+                {' '}
+                <i className="icon-dropdown" />
+              </button>
+              <ul className="dropdown-menu dropdown-menu-right">
```

Comment:
I'd swap "remove member" and "manage groups", the most dangerous action "remove member" should go last